slides/intro-01-introduction.qmd

```diff
@@ -71,9 +71,10 @@ Many thanks to Davis Vaughan, Julia Silge, David Robinson, Julie Jung, Alison Hi
 - Your data budget
 - What makes a model
 - Evaluating models
+- Outlook: Tuning models
```

Comment:
I tripped up a bit on "Outlook" but I'm not sure I have a better recommendation. Emil, if this feels fine to you, feel free to Resolve. :)

Reply:
I'm not a big back of "Outlook" here, since the other bullets don't have a "prefix" type thing